packages/elements-core/src/components/MarkdownViewer/CustomComponents/ReactRouterLink.tsx

```diff
@@ -8,7 +9,7 @@ export const ReactRouterMarkdownLink = ({
   to,
   href: _href,
   children,
-}: JSX.IntrinsicElements['a'] & { to?: string }) => {
+}: Omit<LinkProps, 'target' | 'rel'> & { to?: string }) => {
```

Comment:
This change wasn't related to fixing the attached issue - it was related to a previous change in markdown-viewer (elements had lagged a little and was back on 5.4.2)